src/services/meetings/meetings-schema.ts

```diff
@@ -0,0 +1,43 @@
+import { Schema } from "mongoose";
+
+import { z } from "zod";
+import { v4 as uuidv4 } from "uuid";
+
+export const CommitteeNames = z.enum([
+    "CONTENT",
+    "CORPORATE",
+    "DESIGN",
+    "DEV",
+    "MARKETING",
+    "OPERATIONS",
+]);
+
+export const meetingView = z.object({
+    meetingId: z.coerce.string().default(() => uuidv4()),
+    committeeType: CommitteeNames,
+    startTime: z.coerce.date(),
+});
+
+export const MeetingSchema = new Schema({
+    meetingId: {
+        type: String,
+        required: true,
+        unique: true,
+        default: () => uuidv4(),
+    },
+    committeeType: {
+        type: String,
+        required: true,
+        enum: CommitteeNames.Values,
+    },
+    startTime: {
+        type: Date,
+        required: true,
+    },
+});
+
+export const meetingValidator = z.object({
+    // meetingId: z.string().uuid(),
```

Comment:
Please call these validators better - have a `createMeetingValidator` and `updateMeetingValidator`. Note that currently the update meeting lets you specify an id an it should not.

Also remove unused comment.